sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.net8.0.cs

```diff
@@ -5471,6 +5534,23 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         public override string ToString() { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct Expand : System.IEquatable<Azure.ResourceManager.Compute.Models.Expand>
```

Comment:
this is a single word name which is not acceptable.
based on where this is used, I think we should rename this to `GetVirtualMachineImagesWithPropertiesExpand`

Reply:
updated   Expand: GetVirtualMachineImagesWithPropertiesExpand